### ISSUE TITLE
Modal dialog is not accessible  (A11y)

### ISSUE THREAD

**Issue by j6design** [sfbrigade/adopt-a-drain#144]
Modal dialog is not accessible ( see https://github.com/gdkraus/accessible-modal-dialog to make accessible)
	1. Modal should have only one close
